File: src/tweet.js
import { TwitterApi } from 'twitter-api-v2';

// Instantiate with desired auth type (here's Bearer v2 auth)
const twitterClient = new TwitterApi({
  bearerToken: process.env.BEARER_TOKEN,
  appKey: process.env.CONSUMER_KEY,
  appSecret: process.env.CONSUMER_SECRET,
  accessToken: process.env.ACCESS_TOKEN,
  accessSecret: process.env.ACCESS_TOKEN_SECRET,
});

const tweet = 'hello, world'

const { data: createdTweet } = await twitterClient.v2.tweet(tweet);
console.log('Tweet', createdTweet.id, ':', createdTweet.text);

File: server/src/tweet.py
import os
import tweepy

access_token = input('access_token>')
access_token_secret = input('access_token_secret>')

client = tweepy.Client(
    os.environ['BEARER_TOKEN'],
    os.environ['CONSUMER_KEY'],
    os.environ['CONSUMER_SECRET'],
    access_token,
    access_token_secret
)

tweet_text = input('text>')

client.create_tweet(text=tweet_text)
